syft/serde/serde.py

```diff
@@ -92,6 +93,82 @@
     _simplify_script_module,
 )
 
+# Maps a type to a tuple containing its simplifier and detailer function.
+# IMPORTANT: keep these structures sorted A-Z (by type name).
+MAP_TO_SIMPLIFIERS_AND_DETAILERS = OrderedDict(
+    {
+        dict: (_simplify_dictionary, _detail_dictionary),
+        list: (_simplify_collection, _detail_collection_list),
+        numpy.ndarray: (_simplify_ndarray, _detail_ndarray),
+        range: (_simplify_range, _detail_range),
+        set: (_simplify_collection, _detail_collection_set),
+        slice: (_simplify_slice, _detail_slice),
+        str: (_simplify_str, _detail_str),
+        torch.device: (_simplify_torch_device, _detail_torch_device),
+        torch.jit.ScriptModule: (_simplify_script_module, _detail_script_module),
+        torch.jit.TopLevelTracedModule: (_simplify_script_module, _detail_script_module),
+        torch.nn.Parameter: (_simplify_torch_parameter, _detail_torch_parameter),
+        torch.Tensor: (_simplify_torch_tensor, _detail_torch_tensor),
+        tuple: (_simplify_collection, _detail_collection_tuple),
+        type(Ellipsis): (_simplify_ellipsis, _detail_ellipsis),
+    }
+)
+
+MAP_TO_FORCE_FULL_SIMPLIFY = set([VirtualWorker])
```

Comment:
Why do you put this one in a set and not the  OBJ_SIMPLIFIER_AND_DETAILERS and following objects ?

Reply:
Oops, my bad. This should be a list as well.